components/Tabs/src/MDCTabBar.m

```diff
@@ -177,7 +185,7 @@ - (void)setBarTintColor:(UIColor *)barTintColor {
     _barTintColor = barTintColor;
 
     // Update background color.
-    _itemBar.backgroundColor = barTintColor;
+    self.backgroundColor = barTintColor;
```

Comment:
This looks like an unrelated change and not related to the safe area. Is there a reason we want to set the background color of the entire view rather than just the itemBar?

Reply:
Yeah, the frame of the MDCTabBar can (and should be) taller than the _itemBar to leave some space for the home indicator.

Reply:
Ack. Make sense to me. Aside from the nits, everything LGTM!